tests/e2e/tests/Map/InfoIconsAreInteractiveInLayerComponent.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { test } from '@playwright/test';
+import DashboardPage from 'Pages/DashboardPage';
+import MapComponent from 'Pages/MapComponent';
+import UserStateComponent from 'Pages/UserStateComponent';
+import { qase } from 'playwright-qase-reporter';
+import { NoTriggerDataSet } from 'testData/testData.enum';
+
+import { FloodsScenario } from '../../../../services/API-service/src/scripts/enum/mock-scenario.enum';
+import {
+  getAccessToken,
+  mockFloods,
+  resetDB,
+} from '../../helpers/utility.helper';
+import LoginPage from '../../Pages/LoginPage';
+
+let accessToken: string;
+
+test.beforeEach(async ({ page }) => {
+  // Login
+  const loginPage = new LoginPage(page);
+  accessToken = await getAccessToken();
+  await resetDB(accessToken);
+
+  // We should maybe create one mock for all different disaster types for now we can just use floods
+  await mockFloods(
+    FloodsScenario.NoTrigger,
+    NoTriggerDataSet.CountryCode,
+    accessToken,
+  );
+
+  await page.goto('/');
+  await loginPage.login(
+    NoTriggerDataSet.UserMail,
+    NoTriggerDataSet.UserPassword,
+  );
+});
+
+test(
+  qase(33, 'Interaction with info icon in map layer component'),
```

Comment:
is it possible and/or interesting and/or feasible to add a link to the qase test case behind the number 33 here (and for all other test cases?). Feel free to say 'good idea, but too much work for now'.

Reply:
I guess it would make sense to add them for all test cases so far I guess, but agree on switching to Azure so good enough for now. But if you feel like it in any of the upcoming items still, go ahead.